src/services/nasa/nasa-asteroids.service.js

```diff
@@ -0,0 +1,48 @@
+import {axiosNasaClient} from "../../clients/index.js";
+
+const parseAsteroidsToArray = (dataArray, dangerousOnly) => {
+    const result = [];
+    dataArray.forEach((data) => {
+        if (!dangerousOnly || data.is_potentially_hazardous_asteroid === true)
+            result.push({
+                id: data.id,
+                name: data.name,
+                estimated_diameter_in_meters: {
+                    estimated_diameter_min: data.estimated_diameter.meters.estimated_diameter_min,
+                    estimated_diameter_max: data.estimated_diameter.meters.estimated_diameter_max
+                },
+                is_potentially_hazardous_asteroid: data.is_potentially_hazardous_asteroid,
+                close_approach_date_full: data.close_approach_data[0].close_approach_date_full,
+                relative_velocity_kilometers_per_second: data.close_approach_data[0].relative_velocity.kilometers_per_second,
+            });
+    })
+    return result;
+}
+
+const parseAsteroidsToCount = (dataArray, dangerousOnly) => {
+    let count = 0;
+    dataArray.forEach((data) => {
+        if (!dangerousOnly || data.is_potentially_hazardous_asteroid === true) {
+            count++
+        }
+    })
+    return {count};
+}
+
+const parseNasaAsteroidsData = (data, countOnly, dangerousOnly) => {
```

Comment:
I'm not sure, but it seems like in case `countOnly === true` we will get array `[count, count, count, ...]`, Correct me if I'm wrong, please

Reply:
In this case we get such response:
`{
    "2024-03-05": {
        "count": 0
    },
    "2024-03-06": {
        "count": 1
    },
    "2024-03-04": {
        "count": 2
    },
    "2024-03-07": {
        "count": 0
    },
    "2024-03-08": {
        "count": 2
    }
}`

Reply:
Got it.
Didn't notice separation by date :)